app/upgrades/v2_0/validator-proofs/verify.py

```
@@ -0,0 +1,69 @@
+    prefix, address_bytes = bech32.bech32_decode(entry["consensus_address"])
+    validator_acc_address = bech32.bech32_encode("kyve", address_bytes)
+
+    verify_tx(api_endpoint, entry["proof_1"], entry["protocol_address"], validator_acc_address)
+    verify_tx(api_endpoint, entry["proof_2"], validator_acc_address, entry["protocol_address"])
+
+
+def verify_network(name, api_endpoint):
+    status = {"correct": 0, "error": 0}
+    for file in os.listdir("./" + name):
+        try:
+            proof = json.load(open("./{}/{}".format(name, file)))
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)